plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/db2/internal/messages.properties

```diff
@@ -15,5 +15,7 @@ DB2WizardPage_installDrivers=Install DB2 drivers from client
 DB2WizardPage_connectionTask=Attempting to make database connection
 DB2WizardPage_warning=An error will occur because the DB2 drivers were not loaded.  Please restart application
 DB2WizardPage_title=DB2 Wizard
+DB2WizardPage_button_lookup_text=Get schemas
```

Comment:
Take a screenshot of the wizard page and store it down below 'docs/user/en/images/db2_page/' as DB2.jpg
Add it to stage and commit it like other sources. That's should be enought.

We can separate the cleanup task for docs from this commit ("Add features ..") and move on ;).